src/qt/peertablemodel.cpp

```diff
@@ -71,6 +71,8 @@ QVariant PeerTableModel::data(const QModelIndex& index, int role) const
         switch (column) {
         case NetNodeId:
             return (qint64)rec->nodeStats.nodeid;
+        case Age:
+            return GUIUtil::FormatPeerAge(rec->nodeStats.m_connected);
```

Comment:
852b036a7cc0d38f00558125e04528d4cd0fa2dd

Following my previous comment, this could be
```cpp
return GUIUtil::FormatAge(m_now - rec->nodeStats.m_connected);
```